qupath-core/src/main/java/qupath/lib/objects/utils/ObjectMerger.java

```diff
@@ -570,6 +680,74 @@ private static PathObject mergeObjects(List<? extends PathObject> pathObjects) {
         return mergedObject;
     }
 
+    private static MeasurementList mergeMeasurements(List<? extends PathObject> pathObjects, MeasurementStrategy measurementStrategy) {
+        if (pathObjects.isEmpty()) {
+            return null;
+        }
+        return switch(measurementStrategy) {
+            case IGNORE -> null;
+            case USE_FIRST -> pathObjects.getFirst().getMeasurementList();
+            case MEAN -> {
+                var out = MeasurementListFactory.createMeasurementList(
+                        pathObjects.getFirst().getMeasurementList().size(),
+                        MeasurementList.MeasurementListType.FLOAT
+                );
+                for (var es: pathObjects.getFirst().getMeasurements().entrySet()) {
```

Comment:
This looks convoluted and inefficient - can you just use `pathObjects.getFirst().getMeasurementList().getNames()`?